music.py

```diff
@@ -0,0 +1,84 @@
+import os
+import collections
+
+from pymongo import MongoClient
```

Comment:
You'll probably have some users complain that pymongo isn't an optional dependency, just FYI.